src/components/ui/PaymentRequestDashboard/Tabs.css

```diff
@@ -0,0 +1,44 @@
+button,
+fieldset,
+input {
+  all: unset;
+}
+
+.TabsList {
+  flex-shrink: 0;
+  display: flex;
+}
+
+.TabsTrigger {
+  color: #73808c;
+  background-color: transparent;
+  padding-top: 0px;
+  height: 5rem;
+  user-select: none;
+}
+
+.TabsTrigger:hover {
+  color: #00264d;
+  cursor: pointer;
+}
+
+.TabsTrigger[data-state='active'] {
+  color: #00264d;
+  background-color: white;
+  box-shadow: inset 0 2px 0 0 #00b2b2;
+  cursor: default;
+}
+
+.TabsTrigger[data-disabled] {
+  pointer-events: none;
```

Comment:
I'll also add `cursor: not-allowed;` so you'd get a nice OS mouse distinction.

Reply:
Here's the Tailwind documentation for it so you can see it: https://tailwindcss.com/docs/cursor